client/src/features/dashboard/components/ProjectsDashboard.tsx

```diff
@@ -0,0 +1,182 @@
+/*!
+ * Copyright 2023 - Swiss Data Science Center (SDSC)
+ * A partnership between École Polytechnique Fédérale de Lausanne (EPFL) and
+ * Eidgenössische Technische Hochschule Zürich (ETHZ).
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { SearchEntitiesQueryParams, useSearchEntitiesQuery } from "../../kgSearch/KgSearchApi";
+import { SortingOptions } from "../../../utils/components/sortingEntities/SortingEntities";
+import { InfoAlert } from "../../../utils/components/Alert";
+import React, { Fragment } from "react";
+import { Docs } from "../../../utils/constants/Docs";
+import { ExternalLink } from "../../../utils/components/ExternalLinks";
+import { Link, useHistory } from "react-router-dom";
+import { urlMap } from "../../../project/list/ProjectList.container";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { Url } from "../../../utils/helpers/url";
+import ListDisplay from "../../../utils/components/List";
+import { Loader } from "../../../utils/components/Loader";
+import { useDispatch } from "react-redux";
+import { setAuthor, setType } from "../../kgSearch/KgSearchSlice";
+import { KgAuthor } from "../../kgSearch/KgSearch";
+import { Button } from "../../../utils/ts-wrappers";
+import useGetRecentlyVisitedProjects from "../../../utils/customHooks/useGetRecentlyVisitedProjects";
+
+interface ProjectAlertProps {
+  total?: number;
+}
+function ProjectAlert({ total }: ProjectAlertProps) {
+  if (total === undefined)
+    return null;
+
+  return total === 0 ?
+    <InfoAlert timeout={0}>
+      <div data-cy="project-alert" className="mb-0" style={{ textAlign: "justify" }}>
+        <h3><strong>You don’t have any project yet.</strong></h3>
```

Comment:
I think it would be a bit better like this:
```suggestion
        <h3><strong>You do not have any projects yet.</strong></h3>
```